recirq/quantum_chess/quantum_board.py

```diff
@@ -374,7 +374,7 @@ def unhook(self, qubit: cirq.Qid) -> cirq.Qid:
 
         # Replace operations using the qubit with the ancilla instead
         self.circuit = self.circuit.transform_qubits(lambda q: new_qubit
-                                                     if q == qubit else q)
+        if q == qubit else q)
```

Comment:
Is this line correct?  It looks weird.

Reply:
Good catch. yapf sometimes goes against PEP just to make it look nice, apparently it failed here. Switched it to a single line.